.changelog/3124.intenral.md

```diff
@@ -0,0 +1 @@
+go/e2e/txsource: Periodically restart runtime nodes
```

Comment:
Fix filename (typo in "internal").

Reply:
:+1: would think the `lint fragments` check catch this :-)